notebooks/DrizzlePac/align_mosaics/requirements.txt

```diff
@@ -1,7 +1,7 @@
-astropy==5.3.3
-astroquery==0.4.6
-drizzlepac==3.5.1
-ipython==8.11.0
-matplotlib==3.7.0
-numpy==1.23.4
-photutils==1.12.0
+astropy>=5.3.3
+astroquery>=0.4.6
+drizzlepac>=3.5.1
+ipython>=8.11.0
+matplotlib>=3.7.0
+numpy>=1.23.4
```

Comment:
this is the only requirements file where I see numpy set to 1.23 instead of 1.26, but I'm not sure why.